src/containerapp/HISTORY.rst

```diff
@@ -7,13 +7,21 @@ upcoming
 
 1.2.0b4
 ++++++
+* 'az containerapp function list': List functions in a container app
+* 'az containerapp function show': Show specific function in a container app
+* 'az containerapp function keys show': Show specific function key in a container app
```

Comment:
`1.2.0b4` has been released, please move these description to `upcoming` part (Line 7).